pennylane/gradients/pulse_generator_gradient.py

```diff
@@ -658,14 +659,20 @@ def circuit(params):
         before the parametrized time evolution. Here, the variable :math:`x` is just a
         convenient way to write the modified cost function. Note that its derivative with
         respect to :math:`x` can be computed with the standard two-term parameter-shift
-        rule for Pauli rotation gates.
+        rule for Pauli rotation gates. I.e.
```

Comment:
Personal preference:
```suggestion
        rule for Pauli rotation gates, i.e.
```